<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="aed">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>566</width>
      <height>663</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/overlay/aedNoElectrode.png);
background-color: rgba(255, 255, 255, 0);</string>
    </property>
   </widget>
   <widget class="QPushButton" name="shockButton">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>280</y>
      <width>131</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/buttons/shockButton.png);
border-radius: 20px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="powerButton">
    <property name="geometry">
     <rect>
      <x>465</x>
      <y>558</y>
      <width>61</width>
      <height>46</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
image: url(:/buttons/powerButton.png);
border-radius: 20px;
}

QPushButton:hover {
image: url(:/buttons/powerbuttonON.png);
border-radius: 20px;
}
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="statusIndicator">
    <property name="geometry">
     <rect>
      <x>54</x>
      <y>558</y>
      <width>100</width>
      <height>46</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../res/aed.qrc">:/indicators/statusOff.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="light6">
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>317</y>
      <width>12</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../res/aed.qrc">:/indicators/lightOff.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="light1">
    <property name="geometry">
     <rect>
      <x>168</x>
      <y>247</y>
      <width>12</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../res/aed.qrc">:/indicators/lightOff.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="light5">
    <property name="geometry">
     <rect>
      <x>179</x>
      <y>409</y>
      <width>12</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../res/aed.qrc">:/indicators/lightOff.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="light4">
    <property name="geometry">
     <rect>
      <x>404</x>
      <y>403</y>
      <width>12</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../res/aed.qrc">:/indicators/lightOff.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="light3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>295</y>
      <width>12</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../res/aed.qrc">:/indicators/lightOff.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="light2">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>142</y>
      <width>12</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="../res/aed.qrc">:/indicators/lightOff.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGraphicsView" name="display">
    <property name="geometry">
     <rect>
      <x>205</x>
      <y>184</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(94, 92, 100);
alternate-background-color: rgb(208, 230, 191);</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="userPanel">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>10</y>
      <width>391</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(154, 153, 150);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <attribute name="title">
      <string>User Panel</string>
     </attribute>
     <widget class="QCheckBox" name="battery">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>81</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Battery</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="childPads">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>190</y>
        <width>114</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Child pads</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="adultPads">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>190</y>
        <width>111</width>
        <height>23</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Adult pads</string>
      </property>
     </widget>
     <widget class="QLabel" name="electrodes_2">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>160</y>
        <width>80</width>
        <height>18</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>17</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Electrodes</string>
      </property>
     </widget>
     <widget class="QLabel" name="userdisplay">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>311</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(220,220,220);
color: black;
text-align: center;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="CPR">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>240</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>CPR</string>
      </property>
     </widget>
     <widget class="QLabel" name="performcpr">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>start/stop CPR on 
       patient</string>
      </property>
     </widget>
     <widget class="QLabel" name="powerLevel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>80</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>100%</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changeBatteryLevelButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>141</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>Change Battery Level</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="batteryLevelInput">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>80</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="inputMethodHints">
       <set>Qt::ImhDigitsOnly</set>
      </property>
      <property name="maximumBlockCount">
       <number>0</number>
      </property>
     </widget>
     <widget class="QCheckBox" name="selfTestCheckbox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>120</y>
        <width>85</width>
        <height>20</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="text">
       <string>Self Test</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QGroupBox" name="rhythmGroupBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>120</y>
        <width>121</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>Rhythm</string>
      </property>
      <widget class="QRadioButton" name="PEA_RadioButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>70</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>PEA</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="Asytole_RadioButton">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Asytole</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="VF_RadioButton">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>30</y>
         <width>51</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>VF</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="VT_RadioButton">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>50</y>
         <width>41</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>VT</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="Regular_RadioButton">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>110</y>
         <width>99</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Regular</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="newRhythmButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>270</y>
        <width>100</width>
        <height>32</height>
       </rect>
      </property>
      <property name="text">
       <string>New Rhythm</string>
      </property>
     </widget>
     <widget class="QSlider" name="cprDepth">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>280</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QSlider::groove:horizontal {
    border: 1px solid #A0A0A0; /* Set the border color of the groove */
    height: 8px; /* Set the height of the groove */
    background: #E0E0E0; /* Set the background color of the groove */
}

QSlider::handle:horizontal {
    background: #505050; /* Set the background color of the handle */
    border: 1px solid #707070; /* Set the border color of the handle */
    width: 14px; /* Set the width of the handle */
}

QSlider::sub-page:horizontal {
    background: #E0E0E0; /* Set the background color of the portion of the groove covered by the handle */
}
</string>
      </property>
      <property name="maximum">
       <number>100</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="performcpr_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>270</y>
        <width>81</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Times New Roman</family>
        <pointsize>15</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>CPR depth</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="displayContainer">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>370</y>
      <width>351</width>
      <height>291</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="shockCount">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>SHOCKS: 00</string>
     </property>
    </widget>
    <widget class="QLabel" name="elapsedtime">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>20</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>00:00</string>
     </property>
    </widget>
    <widget class="QLabel" name="heartSignal">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>271</width>
       <height>131</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="voiceprompt">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>200</y>
       <width>281</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>20</pointsize>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="heartbeat">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>150</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>17</pointsize>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="barGraph">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>51</width>
       <height>151</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <zorder>display</zorder>
   <zorder>aed</zorder>
   <zorder>shockButton</zorder>
   <zorder>powerButton</zorder>
   <zorder>statusIndicator</zorder>
   <zorder>light6</zorder>
   <zorder>light1</zorder>
   <zorder>light5</zorder>
   <zorder>light4</zorder>
   <zorder>light3</zorder>
   <zorder>light2</zorder>
   <zorder>userPanel</zorder>
   <zorder>displayContainer</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../res/aed.qrc"/>
  <include location="../res/aed.qrc"/>
 </resources>
 <connections/>
</ui>
